packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -25,3 +25,28 @@ headingLevels.forEach((headingLevel) => test(`should render the correct heading
     // h2 is the default / fallback value
     await expect(component.locator('h2.c-modal-heading')).toContainText(props.heading);
 }));
+
+test.describe('`When the Pie Modal is closed`', () => {
+    test.skip('should dispatch event `pie-modal-close`', async ({ mount }) => {
```

Comment:
Will come back to this test in my next pr. Locally it all passes but on CI it fails due to a timeout.